src/Compilers/CSharp/Portable/Symbols/AssemblySymbol.cs

```diff
@@ -912,63 +912,9 @@ private static bool IsAcceptableMatchForGetTypeByMetadataName(NamedTypeSymbol ca
         /// </summary>
         public abstract AssemblyMetadata? GetMetadata();
 
-        INamedTypeSymbol? IAssemblySymbol.ResolveForwardedType(string fullyQualifiedMetadataName)
```

Comment:
Why is all this code being deleted in an annotation PR?

Reply:
It isn't.